Neos.Flow.Log/Classes/Backend/FileBackend.php

```diff
@@ -241,7 +241,7 @@ public function append(string $message, int $severity = LOG_INFO, $additionalDat
         }
         $ipAddress = ($this->logIpAddress === true) ? str_pad(($_SERVER['REMOTE_ADDR'] ?? ''), 15) : '';
         $severityLabel = $this->severityLabels[$severity] ?? 'UNKNOWN  ';
-        $output = (new \DateTime())->format('y-m-d H:i:s') . $processId . ' ' . $ipAddress . $severityLabel . ' ' . str_pad((string)$packageKey, 20) . ' ' . $message;
+        $output = (new \DateTime())->format('y-m-d H:i:s') . $processId . ' ' . $ipAddress . ' ' . $severityLabel . ' ' . str_pad((string)$packageKey, 20) . ' ' . $message;
```

Comment:
As the IP address is optional, we should rather add the space after the str_pad, instead of the $output.

Reply:
I rebased the patch and changed the location of the space to be added in line 242 when $ipAddress is populated.